--- czmq-4.2.0/Makefile.am.orig	2020-04-07 19:34:47.841128420 +0200
+++ czmq-4.2.0/Makefile.am	2020-04-07 19:35:40.744175153 +0200
@@ -15,7 +15,7 @@
     ${libmicrohttpd_CFLAGS} \
     -I$(srcdir)/include
 
-project_libs = ${libzmq_LIBS} ${uuid_LIBS} ${systemd_LIBS} ${lz4_LIBS} ${libcurl_LIBS} ${libmicrohttpd_LIBS}
+project_libs = ${libzmq_LIBS} ${uuid_LIBS} ${systemd_LIBS} ${lz4_LIBS} ${libcurl_LIBS} ${libmicrohttpd_LIBS} -lpthread
 
 SUBDIRS = doc
 SUBDIRS += include
